seerpy/auth.py

```diff
@@ -204,6 +215,42 @@ def _read_cookie(self):
                 self.cookie = json.loads(f.read().strip())
 
 
+class SeerApiKeyAuth(BaseAuth):
+    """
+    Creates an authenticated connection to the Seer API using an API key. This will become the
+    default for most use cases.
+    """
+    def __init__(self, api_key_id, api_key_path, api_url=None):
+        """
+        Authenticate session using API key
+
+        Parameters
+        ----------
+        api_key_id : str
+            The UUID for a Seer api key
+        api_key_path : str
+            The path to a Seer api key file
+        api_url : str, optional
+            Base URL of API endpoint
+        """
+
+        # TODO: not sure what the api url actually is
+        super(SeerApiKeyAuth,
+              self).__init__(api_url if api_url is not None else "https://sdk.seermedical.com/api")
```

Comment:
Could this be simplified to the following (and in SeerAuth above) by setting `api_url='https://......'` in the function args?

```suggestion
              self).__init__(api_url)
```

Reply:
As it turns out, no - you did it that way for a reason!